extensions/wikia/SpecialPromote/SpecialPromoteHelper.class.php

```diff
@@ -479,16 +479,16 @@ protected function checkWikiStatus($wikiId, $langCode) {
 		}		
 		foreach($imageStatuses as $status) {
 			switch($status) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)